src/dotnet/commands/dotnet-test/LocalizableStrings.cs

```diff
@@ -36,7 +36,7 @@ internal class LocalizableStrings
 
         public const string CmdConfiguration = "CONFIGURATION";
 
-        public const string CmdConfigDescription = "Configuration to use for building the project.  Default for most projects is  \"Debug\".";
+        public const string CmdConfigDescription = "Configuration to use for building the project.  Default for most projects is \"Debug\".";
```

Comment:
I find this string "Default for most projects" which is used in a lot of places  really confusing for customers. It doesn't tell me when I would not get the default.